tabular/src/autogluon/tabular/configs/hyperparameter_configs.py

```diff
@@ -18,12 +18,12 @@
         'RF': [
             {'criterion': 'gini', 'ag_args': {'name_suffix': 'Gini', 'problem_types': ['binary', 'multiclass']}},
             {'criterion': 'entropy', 'ag_args': {'name_suffix': 'Entr', 'problem_types': ['binary', 'multiclass']}},
-            {'criterion': 'mse', 'ag_args': {'name_suffix': 'MSE', 'problem_types': ['regression']}},
+            {'criterion': 'mse', 'ag_args': {'name_suffix': 'MSE', 'problem_types': ['regression', 'quantile']}},
         ],
         'XT': [
             {'criterion': 'gini', 'ag_args': {'name_suffix': 'Gini', 'problem_types': ['binary', 'multiclass']}},
             {'criterion': 'entropy', 'ag_args': {'name_suffix': 'Entr', 'problem_types': ['binary', 'multiclass']}},
-            {'criterion': 'mse', 'ag_args': {'name_suffix': 'MSE', 'problem_types': ['regression']}},
+            {'criterion': 'mse', 'ag_args': {'name_suffix': 'MSE', 'problem_types': ['regression', 'quantile']}},
```

Comment:
@Innixma For XT and RF: maybe we don't need the MSE name_suffix for `problem_type in [QUANTILE, REGRESSION], since there is only ever one split criterion being used?

This suffix may appear a bit confusing for QUANTILE (even though it still is technically the split criterion)

Reply:
I'm ok with the name_suffix being removed. I would say that changing model names shouldn't be considered a breaking change.

Reply:
Should I leave it as is? or remove 'name_suffix' in this PR?

Reply:
lets keep it for now so we don't alter functionality outside of quantile. We can do it in a separate PR, I think it's low in priority.